### DISCUSSION TITLE
I got keep rejecting from GitHub student pack

### DISCUSSION THREAD

**Discussion by mefeozh** [community/community#51326] [GitHub Education]
I have downloaded my transcript file two times, and it clearly says I am an active student at my university, but I keep rejecting it from the GitHub student pack. The problem is GitHub only accepts JPEG files, while my transcript is a PDF file. Because of that, I needed to change my pdf file to a jpeg file. The funny part is after I summited the file, it said, "your file comes from a third party."  I personally don't get why anyone would use only a jpeg file. My problem is I can't identify myself to GitHub to get student pack, and I  do not know how to identify myself to GitHub.

**Comment by samus-aran** [comment]
Hi @mefeozh 👋 

Based on the rejection reason we would recommend that you obtain another form of evidence to submit as it implies that something on it has been detected previously in fraudulent applications. 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

We would also suggest that if you have recently updated your Profile details to apply that you please log out and log back into GitHub before re-applying 👍 

Let us know how you get on.
